resources/services/emr/block_public_access_configs.go

```diff
@@ -110,6 +110,10 @@ func fetchEmrBlockPublicAccessConfigs(ctx context.Context, meta schema.ClientMet
 		options.Region = c.Region
 	})
 	if err != nil {
+		if client.IgnoreNotAvailableRegion(err) {
+			meta.Logger().Debug("received InvalidRequestException on GetBlockPublicAccessConfiguration, api is not available in the current Region.", "region", c.Region, "err", err)
```

Comment:
```suggestion
			meta.Logger().Debug("received InvalidRequestException on GetBlockPublicAccessConfiguration, api is not available in the current Region.", "err", err)
```
since region is apparently in the logger anyway, I removed it